pkg/new-ui/v1beta1/backend.go

```diff
@@ -574,3 +586,144 @@ func (k *KatibUIHandler) FetchTrial(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+// FetchTrialLogs fetches logs for a trial in specific namespace.
+func (k *KatibUIHandler) FetchTrialLogs(w http.ResponseWriter, r *http.Request) {
+	trialName := r.URL.Query()["trialName"][0]
+	namespace := r.URL.Query()["namespace"][0]
```

Comment:
@d-gol Just a minor comment here. Maybe add checks here similar to what we do [here](https://github.com/d-gol/katib/blob/4afe97603ef2ffbe868c42371cf6716e431754bd/pkg/new-ui/v1beta1/hp.go#L196-L213)